### DISCUSSION TITLE
Reply a tweet

### DISCUSSION THREAD

**Discussion by kevinjamau** [noweh/twitter-api-v2-php#19] [Q&A] [answered]
How we can reply a tweet by it's Id?

**Comment by noweh** [comment, marked as answer]
Hello!
It's possible with something like that:
`$return = (new TwitterClient($twitterSettings))->tweet()->performRequest('POST', [
                'text' => 'my answer',
                'reply' => ['in_reply_to_tweet_id' => 'TweetId']
]);`
